### PR TITLE
Bottom navigation improvements

### PR DESCRIPTION
**Pull request by NovacAndrei**

### CODE REVIEW

File: src/screens/dashboard/dashboard.tsx [hunk 1]
```diff
@@ -140,19 +148,60 @@ export class DashboardScreenComponent extends React.Component<
...
-    public async componentDidMount() {
+    public componentDidMount() {
         this.props.navigation.setParams({
             setDashboardMenuBottomSheet: this.setDashboardMenuBottomSheet
         });
     }
 
+    public componentDidUpdate(prevProps: IReduxProps) {
+        if (this.props.selectedBlockchain !== prevProps.selectedBlockchain) {
+            const found =
```
Comment (krisboit) [inline src/screens/dashboard/dashboard.tsx]: `found` is too confusing as a name for this context :)) you are basically verifying if something is not found ...

Comment (NovacAndrei) [reply src/screens/dashboard/dashboard.tsx]: Forgot about this :)) Sorry, fixing :)

File: src/components/bottom-sheet/blockchain-navigation-bottom-sheet/blockchain-navigation-bottom-sheet.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    public renderToken = (blockchain: Blockchain) => {
+        const { styles, theme } = this.props;
+
+        const coin = getBlockchain(blockchain).config.coin;
+        const logo = getBlockchain(blockchain).config.tokens[coin].logo;
+
+        return (
+            <TouchableOpacity
```
[the remark this answers is not in the document]
Comment (NovacAndrei) [reply src/components/bottom-sheet/blockchain-navigation-bottom-sheet/blockchain-navigation-bottom-sheet.tsx]: fixed the import

File: src/screens/dashboard/dashboard.tsx (hunk 1 above)
Comment (krisboit) [reply src/screens/dashboard/dashboard.tsx]: also you can use indexOf to check if an element is in an array :P

Comment (NovacAndrei) [reply src/screens/dashboard/dashboard.tsx]: yes, much easier with `indexOf`, fixed :D

**Pull request merged**